### ISSUE TITLE
OCPCalculator error

### ISSUE THREAD

**Issue by iMasak** [M-asaki-K/OCPmodels_seminar#2]
I have an error on this command:
```
#Define the calculator
calc = OCPCalculator(config_yml=config_yml_path, checkpoint=checkpoint_path, cpu = False)
```
I corrected; "checkpoint=checkpoint_path" --> "checkpoint_path=checkpoint_path", and it worked.
Is it due to a difference in the version of OCPCalculator or so?
thank you,

**Comment by M-asaki-K** [comment]
Thank you for your question. 

As you pointed out, the cause of the error is due to a version difference in OCP. The notebook I created was based on the OCP tutorial (#314, https://github.com/Open-Catalyst-Project/ocp/blob/2ffe8bc93416b51908247a5bad655df10f65636c/tutorials/OCP_Tutorial.ipynb). I would like to note that the version I referred to was from the end of 2022.

`
calc = OCPCalculator(config_yml=config_yml_path, checkpoint=checkpoint_path)
`

Following an update, the argument name in OCPCalculator has been changed from 'checkpoint' to 'checkpoint_path'.
